runner/copy_test.go

```diff
@@ -791,6 +791,10 @@ func (m MockOfflineTable) Write(provider.ResourceRecord) error {
 	return nil
 }
 
+func (m MockOfflineTable) WriteBatch([]provider.ResourceRecord) error {
+	return nil
```

Comment:
Have we just continued to not implement thing and return nil? Does this thing get used?